### ISSUE TITLE
Loading construction issue with labels and polygon

### ISSUE THREAD

**Issue by dickinson0718** [dulimarta/spherical-easel#83]
When loading a construction that contains a measured polygon (like "Test Polygon 5"), the shading and the labels are not displayed correctly when first loaded:
![Screenshot 2024-07-25 at 2 04 55 PM](https://github.com/user-attachments/assets/ec8efbbb-cbfd-40fd-8a54-c37521413076)
If you move a point (or anything else) the shading and the labels are correctly displayed for anything that was moved:
![Screenshot 2024-07-25 at 2 07 06 PM](https://github.com/user-attachments/assets/b223f900-cdcf-497d-8cc5-7f4d8cafc193)

**Comment by dulimarta** [comment]
This is partially fixed:

- Labels (except for angle markers) show up in the correct spot
- The polygon boundary lines do not show up yet

**Comment by dulimarta** [comment]
Issue with angle markers is fixed in d141fce9

**Comment by dulimarta** [comment]
The issue is caused by TwoJS update cycle not being invoked when interpreting/loading a command script.
The polygon shading depends on its boundary segment paths have their vertices computed correctly (by TwoJS). But due to single-threaded nature of JS engine, the TwoJS update() function is not invoked when `CommandInterpreter::interpret` is executing. This issue is fixed by forcing a TwoJS update

```
const updateTwoJS = new UpdateTwoJSCommand()
command
  .forEach((c: string) => {
    group.addCommand(executeIndividual(c));
    group.addCommand(updateTwoJS) //<=== Adding this command fixes the issue
  });
group.execute();

```
